LiverSegments/Logic/vtkLiverSegmentsLogic.cxx

```diff
@@ -0,0 +1,133 @@
+ /*===============================================================================
+
+   Project: LiverSegments
+   Module: vtkSegmentClassificationLogic.cxx
+
+   Copyright (c) 2019,  Oslo University Hospital
+
+   All rights reserved. This is propietary software. In no event shall the author
+   be liable for any claim or damages 
+
+   ===============================================================================*/
```

Comment:
Please, check the license text in any of the other files and change it accordingly